### PR TITLE
Report available condition regardless of which template versions were deployed

### PR DESCRIPTION
**Pull request by omeryahud**
**What this PR does / why we need it**:
Previously, all templates were aligned to the same version, but a previous fix (https://github.com/kubevirt/kubevirt-ssp-operator/pull/253)

locked deprecated templates to older versions, so the operator did not report the Available condition.

This fix makes the operator report Available when all templates were deployed, regardless of their version.

Signed-off-by: Omer Yahud <oyahud@redhat.com>

**Which issue(s) this PR fixes** 
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1904830

**Release note**:

```release-note
None
```

### CODE REVIEW

File: roles/KubevirtCommonTemplatesBundle/tasks/main.yml [hunk 1]
```diff
@@ -69,23 +69,19 @@
...
-
 - name: "Get all templates"
   set_fact:
-    deployed_templates_after: "{{ lookup('k8s', api_version=ct_status.result.results[0].result.apiVersion, kind='template', label_selector=label) | length }}"
+    deployed_templates_after: "{{ lookup('k8s', api_version=ct_status.result.results[0].result.apiVersion, kind='template', label_selector='template.kubevirt.io/type=base') | length }}"
 
 - name: "Set Available status"
   set_fact:
-    available: "{{ true if new_templates <= deployed_templates_after else false }}"
+    available: "{{ True if new_templates <= deployed_templates_after else False }}"
```
Comment (sarahbx) [inline roles/KubevirtCommonTemplatesBundle/tasks/main.yml:84]: Ansible doesn't particularly care about True vs true, but yaml linter may. Why the change?

Comment (omeryahud) [reply roles/KubevirtCommonTemplatesBundle/tasks/main.yml:84]: Just for consistency really, nothing more

**Comment by omeryahud** [comment]
I've updated the templates deployment tests to wait for the condition (should have done it a long time ago, but missed it)

**Comment by omeryahud** [comment]
Functests are failing due to a flakey test I am addressing the issue in a different PR https://github.com/kubevirt/kubevirt-ssp-operator/pull/256

File: roles/KubevirtCommonTemplatesBundle/tasks/main.yml (hunk 1 above)
Comment (dankenigsberg) [reply roles/KubevirtCommonTemplatesBundle/tasks/main.yml:84]: for a layperson like myself, it is important to keep cosmetics like this on another PR.
